packages/frontend/src/components/Trade/YourVaults.tsx

```diff
@@ -0,0 +1,53 @@
+import LabelWithTooltip from '../LabelWithTooltip'
+import SqueethCard from '../SqueethCard'
+import useYourVaults from '../../hooks/useYourVaults'
+import { Grid, Typography } from '@material-ui/core'
+import { toTokenAmount } from '../../utils/calculations'
+import BigNumber from 'bignumber.js'
+import Link from 'next/link'
+import { FC } from 'react'
+
+const YourVaults: FC = () => {
+  const { data: { vaults } = {}, loading, error } = useYourVaults()
+
+  if (error) {
+    return <Typography color="error">{error.message}</Typography>
+  }
+
+  if (loading) {
+    return <Typography>Loading...</Typography>
+  }
+
+  return (
+    <>
+      {vaults?.map((vault, index) => (
+        <Link key={vault.id} href={`/vault/${vault.id}`} passHref>
```

Comment:
Can we add labels in line instead of in the tooltip?

<img width="908" alt="Screen Shot 2022-03-03 at 8 35 36 PM" src="https://user-images.githubusercontent.com/13340486/156700571-440cf203-a854-4ce5-b13d-2369db925757.png">